scripts/update-apk.sh

```diff
@@ -11,12 +11,28 @@ if [ "$TRAVIS_PULL_REQUEST" != "false" -o "$TRAVIS_REPO_SLUG" != "fossasia/open-
     exit 0
 fi
 
+mkdir $HOME/daily/
+cp -R /home/travis/build/fossasia/open-event-orga-app/app/build/outputs/apk/app-debug.apk $HOME/daily/
+cd $HOME
+
 git clone --quiet --branch=apk https://the-dagger:$GITHUB_API_KEY@github.com/fossasia/open-event-orga-app apk > /dev/null
-cp platforms/android/build/outputs/apk/android-debug.apk apk/test-android-debug.apk
 cd apk
+cp -Rf $HOME/daily/*  ./
+mv app-debug.apk sample-apk-${TRAVIS_BRANCH}.apk
+echo $TRAVIS_COMMIT > meta/deployment/commit_hash
```

Comment:
These are not required for this. This is used only in open-event-android for deploying the generator to kubernetes.